docs/DTOmodels.md

```diff
@@ -58,7 +58,7 @@ description: string
 time: number
 tags: string[]
 
-CurrentProblemDTO (subCollection of DayReport)
+ProblemDTO (subCollection of DayReport)
```

Comment:
strange description format, let's discuss it

Reply:
Don't know how to describe in better way

Reply:
Ask somebody what does it mean. And if the answer will be right you can change nothing)

Reply:
Deleted  text in braces and added described inside DayReport fields